<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlvMaaritys</class>
 <widget class="QWidget" name="AlvMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Verokausi</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="kausiCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Viimeisin ilmoitus</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="viimeisinEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Seuraava ilmoitus</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="seuraavaLabel">
       <property name="text">
        <string>xxx</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Eräpäivä</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="erapaivaLabel">
       <property name="text">
        <string>xxx</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="tilitaNappi">
       <property name="text">
        <string>Tee alv-tilitys</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="ilmoituksetView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="tilitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Näytä tilitys</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="erittelyNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Näytä erittely</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
